funkcje.R

```diff
@@ -464,30 +464,26 @@ wektor_lemy <- f_lematyzacja_wektora_zdan(wektor_czysty, "slownik2.txt")
 
 
 # ------------------------------------------------------------------------------------------------------------------
-# Funkcja oblicza odelosci miedzy dokumentami
+# Funkcja oblicza odelosci miedzy dokumentami - do grupowania hierarchicznego
```

Comment:
Dorzucę do readme definicję grupowania hierarchicznego i niehierarchicznego for dummies